pkg/config/config.go

```diff
@@ -343,6 +358,18 @@ func (dc *DeploymentConfig) addModuleConnection(ref reference, kind connectionKi
 	return nil
 }
 
+// GetModuleConnections returns the graph of connections between modules and
+// from modules to deployment variables
+func (dc *DeploymentConfig) GetModuleConnections() map[string][]ModConnection {
+	return dc.moduleConnections
```

Comment:
Is `moduleConnection` always initialized? it will be nice to have a guarantee that this function never returns `nil`.

Reply:
Yes, it is part of `config.NewDeploymentConfig` which called by both `create` and `expand` directly.